INSTALL.md

```diff
@@ -148,6 +146,11 @@ cd gym-gazebo2
 echo "source `pwd`/provision/mara_setup.sh" >> ~/.bashrc
 source ~/.bashrc
 ```
+**Note**: In Dashing we need to use opensplice implementation of DDS, since Fast-RTPS and others are still buggy and not supported well in this use case. Please export the OpenSplice DDS implementation manually or use the provisioning script before running/training any example of the MARA enviroment.
+
+```sh
+export RMW_IMPLEMENTATION=rmw_opensplice_cpp
+```
```

Comment:
We can add it directly to the code in utils, same as when we launch it on the [real robot](https://github.com/AcutronicRobotics/gym-gazebo2/blob/dashing/gym_gazebo2/utils/ut_launch.py#L156).

Reply:
its already added to the provisioning script

Reply:
Yes it is added in the provisioning script but as an extra precaution we also add it in install and readme.